main.tf

```diff
@@ -121,7 +121,9 @@ resource "aws_security_group_rule" "ingress_cidr_blocks" {
 }
 
 data "aws_ami" "eks_worker" {
-  count = "${var.enabled == "true" && var.image_id == "" ? 1 : 0}"
+  count       = "${var.enabled == "true" && var.image_id == "" ? 1 : 0}"
+  most_recent = true
+  name_regex  = "^amazon-eks-node-[1-9,\\.]+-v\\d{8}$"
```

Comment:
let's parameterize this because it could be changed in the future.
please add a new var `eks_worker_ami_name_regex` with the default value `"^amazon-eks-node-[1-9,\\.]+-v\\d{8}$"`

and rebuild README again after that.

Reply:
done